src/libmongoc/doc/installing.rst

```diff
@@ -141,162 +186,133 @@ Additionally, it is possible to build only libbson by setting the ``-DENABLE_MON
 
-  $ cmake -DENABLE_MAN_PAGES=ON -DENABLE_HTML_DOCS=ON ..
-  $ make mongoc-doc
+.. parsed-literal::
 
-To build only the libbson documentation:
+  $ cd mongo-c-driver-|version|
+  $ cmake -G "Visual Studio 14 2015 Win64" \\
+      "-DCMAKE_INSTALL_PREFIX=C:\\mongo-c-driver" \\
+      "-DCMAKE_PREFIX_PATH=C:\\mongo-c-driver" \\
+      ..
 
-.. code-block:: none
+(Run ``cmake -LH ..`` for a list of other options.)
 
-  $ cmake -DENABLE_MAN_PAGES=ON -DENABLE_HTML_DOCS=ON ..
-  $ make bson-doc
+To see a complete list of the CMake generators available on your specific system, use a command like this:
 
-The ``-DENABLE_MAN_PAGES=ON`` and ``-DENABLE_HTML_DOCS=ON`` can also be added as options to a normal build from a release tarball or from git so that the documentation is built at the same time as other components.
+.. parsed-literal::
+
+ $ cmake --help
 
-Building on macOS
+Executing a build
 -----------------
 
-Install the XCode Command Line Tools::
+Building on Unix, macOS, and Windows (MinGW-W64 and MSYS2)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  $ xcode-select --install
+.. parsed-literal::
 
-The ``cmake`` utility is also required. First `install Homebrew according to its instructions <https://brew.sh/>`_, then::
+  $ cmake --build .
+  $ sudo cmake --build . --target install
 
-  $ brew install cmake
+(Note that the ``sudo`` command may not be applicable or available depending on the configuration of your system.)
 
-Download the latest release tarball:
+In the above commands, the first relies on the default target which builds all configured components.  For fine grained control over what gets built, the following command can be used (for Ninja and Makefile-based build systems) to list all available targets:
 
 .. parsed-literal::
 
-  $ curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/x.y.z/mongo-c-driver-x.y.z.tar.gz
-  $ tar xzf mongo-c-driver-x.y.z.tar.gz
-  $ cd mongo-c-driver-x.y.z
+  $ cmake --build . help
+
+Building on Windows with Visual Studio
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once the project files are generated, the project can be opened directly in Visual Studio or compiled from the command line.
 
-Build and install the driver:
+Let's build using the CMake build tool mode:
 
 .. code-block:: none
 
-  $ mkdir cmake-build
-  $ cd cmake-build
-  $ cmake -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF ..
-  $ make install
+  $ cmake --build . --config RelWithDebInfo
 
-All of the same variations described above (e.g., building only libbson, building documentation, etc.) are available when building on macOS.
+Visual Studio's default build type is ``Debug``, but we recommend a release build with debug info for production use. Now that libmongoc and libbson are compiled, let's install them. Components will be installed to the path specified by ``CMAKE_INSTALL_PREFIX``.
```

Comment:
This "we" seems ok, since this is our recommendation, not something we're telling the reader to do. Remove the "let's".

Reply:
Done.